fuse_variables/src/orientation_2d_stamped.cpp

```diff
@@ -34,12 +34,15 @@
 #include <fuse_core/uuid.h>
 #include <fuse_variables/orientation_2d_stamped.h>
 
+#include <boost/core/demangle.hpp>
 #include <ceres/autodiff_local_parameterization.h>
 
 
 namespace fuse_variables
 {
 
+const std::string Orientation2DStamped::TYPE = boost::core::demangle(typeid(Orientation2DStamped).name());
```

Comment:
I added this static constant to the variable type...which I wish was easier to do.